server/routes/creatingReports/incidentDetails.test.ts

```diff
@@ -256,6 +291,7 @@ describe('POST save and return to tasklist', () => {
       })
   })
 
+  // is saving without incident date correct? - . Dev returns validation error
```

Comment:
worth pointing out and ticketing up for further investigation... good not to leave this kind of comment as it will just provide uncertainty for future maintainers

Reply:
ok